config/routes.rb

```diff
@@ -2,6 +2,15 @@
 require 'sidekiq/cron/web'
 
 Rails.application.routes.draw do
+  Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+    # Protect against timing attacks:
+    # - See https://codahale.com/a-lesson-in-timing-attacks/
+    # - See https://thisdata.com/blog/timing-attacks-against-string-comparison/
```

Comment:
This isn't resolving for me. Is it a dead link? I could find a snapshot that is live: https://web.archive.org/web/20180709235757/https://thisdata.com/blog/timing-attacks-against-string-comparison/ - perhaps we could link to that?